qiskit/utils/mitigation/_filters.py

```diff
@@ -229,7 +228,7 @@ class TensoredFilter:
         since="0.24.0",
         additional_msg="For code migration guidelines, visit https://qisk.it/qi_migration.",
     )
-    def __init__(self, cal_matrices: np.matrix, substate_labels_list: list, mit_pattern: list):
+    def __init__(self, cal_matrices: np.matrix, substate_labels_list: list[str], mit_pattern: list):
```

Comment:
The docstring indicates that it's not just a list of strings:
> a list of the states (as strings, states in the subspace)

Reply:
As far as I understand this means "as strings (states in the subspace)" not "as strings or states in the subspace"